themes/bootstrap3/templates/record/ajaxview-accordion.phtml

```diff
@@ -6,11 +6,11 @@
   $idSuffixEsc = $this->escapeHtmlAttr(md5($this->driver->getUniqueId() . '|' . $this->driver->getSourceIdentifier()));
 ?>
 
-<div class="list-tabs panel-group" id="accordion_<?=$idSuffixEsc?>">
+<div class="list-tabs panel-group" id="accordion_<?=$idSuffixEsc?>" role="tablist">
```

Comment:
It could be debatable whether these really are like tabs, but I'm not sure this functionality is used anywhere and these changes make validation pass with sandal5.